tools/bigwig_outlier_bed/.shed.yml

```
@@ -0,0 +1,20 @@
+categories:
+- Sequence Analysis
+description: pybigtools and numpy code to find continuous runs above a high or below a low quantile cutpoint in bigwig files
+homepage_url: https://github.com/fubar2/bigwig_outlier_bed
```

```suggestion
homepage_url: https://github.com/galaxyproject/tools-iuc/tree/master/tools/bigwig_outlier_bed
```

This should point to https://github.com/jackh726/bigtools but this wrapper is not really for https://github.com/jackh726/bigtools ... its just using it. So we use the IUC URL since the code lives here.